### ISSUE TITLE
Ghost of a Tale (417290)

### ISSUE THREAD

**Issue by Somamint** [ValveSoftware/Proton#2427]
# Compatibility Report
- Name of the game with compatibility issues: Ghost of a Tale
- Steam AppID of the game: 417290

Can't see the Character at all, games freezes after quit

## System Information
- GPU: <Intel Core i7-8750H @ 2.20GHz>
- Driver/LLVM version: <NVIDIA 418>
- Kernel version: <4.18.0-15-generic>
- Link to full system information report as [Gist](https://gist.github.com/):
- Proton version: 3.16-8

## I confirm:
- [X ] that I haven't found an existing compatibility report for this game.
- [X ] that I have checked whether there are updates for my system available.

## Symptoms 
#

**Comment by kisak-valve** [comment]
Hello @Somamint, please copy your system information from Steam (`Steam` -> `Help` -> `System Information`) and put it in a [gist](https://gist.github.com/), then include a link to the gist in this issue report. Also, please add [`PROTON_LOG=1 %command%`](https://github.com/ValveSoftware/Proton#runtime-config-options) to the game's launch options and drag and drop the generated $HOME/steam-$APPID.log into the comment box.

**Comment by Somamint** [comment]
Hello i found a sollution!!! Where can i post my sollutions?
 

Gesendet: Montag, 18. März 2019 um 20:39 Uhr
Von: "kisak-valve" <notifications@github.com>
An: ValveSoftware/Proton <Proton@noreply.github.com>
Cc: Somamint <123Matze-M@web.de>, Mention <mention@noreply.github.com>
Betreff: Re: [ValveSoftware/Proton] Ghost of the Tale (417290) (#2427)


Hello @Somamint, please copy your system information from Steam (Steam -> Help -> System Information) and put it in a gist, then include a link to the gist in this issue report. Also, please add PROTON_LOG=1 %command% to the game's launch options and drag and drop the generated $HOME/steam-$APPID.log into the comment box.

**Comment by ziabice** [comment]
This game works perfectly on my system with Proton 3.16-8 Beta (but even with previous versions of 3.16 branch). The only problem it has is that sometimes you loose the mouse: you have to press Alt-tab to let it work again.

I'm using an AMD RX580 card with Mesa 18.3.4.

[My system specs](https://gist.github.com/ziabice/8c19008e58f82b16a62f7ab65ba235d4#file-sysinfo-txt)

@kisak-valve the correct name of the game is "Ghost of a Tale"

**Comment by Somamint** [comment]
I got the game running with Protontricks and Vulkan:

So after a few tweaks the games running 60 fps with Vulkan!!!

Here is how i dit it:
You need first Protontricks:
https://github.com/Matoking/protontricks

The i open the Terminal an looked for the game :

[code]

protontricks -s Ghost of a tale

[/code]

something like this should come up :


[code]
Found the following games:
Ghost of a Tale (417290)
[/code]


now you open the Gui:

[code]
protontricks 417290 --gui
[/code]

Select the Default Wineprefix


Install a Windows DLL 

[code]
Install here : dotne40 and vcrun2010
[/code]

After installed start the game with 

[code]
"PROTON_NO_ESYNC=1  DefaultLimitNOFILE=1048576 DXVK_HUD=1 %command%"
[/code]

and Your done :)

A good Video who helped me was :

https://www.youtube.com/watch?v=zUJz-SdWul4


But: Mouse starts not working after a while and will only spin left and right, but up and down.
Anyone a sollution?

> [My system specs](https://gist.github.com/Somamint/65f668d5b6fe5897450304e1b9b48e40)

**Comment by ziabice** [comment]
> 
> But: Mouse starts not working after a while and will only spin left and right, but up and down.
> Anyone a sollution?
> 

Simply try to press Alt+Tab then return to the game window.

**Comment by CubeTheThird** [comment]
I have a strange issue with this game where, on starting it up, it will either be running at full performance in the menu (144 FPS in my case), and other times it will be incredibly slow (~15 FPS), and this is in the main menu. This will seem to dictate whether the game's overall performance will be decent, or like a potato.

I'm using the latest proton (6.3-8) with Esync disabled. Tried both with and without gamemoderun, with no difference.

**Comment by SlugSleepy** [comment]
hello, the game does not run on my system, the game crash suddenly and randomly .. when i launch the game, the game crash, sometimes, the game crash when launching game, others time it crash when changing resolution (because yes, the game doesn't want to be fullscreen 1080p)

specs : https://gist.github.com/SlugSleepy/29f94613f8922f1d3f82bcb816c9b358


uploaded crash log with PROTON_LOG=1 %command% 
[steam-417290.log](https://github.com/ValveSoftware/Proton/files/9542999/steam-417290.log)

**Comment by SlugSleepy** [comment]
hello, the game does not run on my system, the game crash suddenly and randomly .. when i launch the game the game crash, sometimes, the game crash when launching game, others time it crash when changing resolution (because yes, the game doesn't want to be fullscreen 1080p)

specs : https://gist.github.com/SlugSleepy/29f94613f8922f1d3f82bcb816c9b358


uploaded crash log with PROTON_LOG=1 %command% 
[steam-417290.log](https://github.com/ValveSoftware/Proton/files/9542999/steam-417290.log)

typo error (tried my best for being readable)

**Comment by SlugSleepy** [comment]
made a video to describe my problem : 
https://www.youtube.com/watch?v=-E_TvgAFyl8

my session becomes unusable after launching the game.
if I move my mouse or press my keyboard, the game crashes faster 
if I reinstall the game, moving the mouse or the keyboard when the game is loading makes the game crash but is only playable in (windowed) mode... which is unpleasant on a small 1080p screen

**Comment by kisak-valve** [comment]
Hello @SlugSleepy, `X connection to :99.0 broken (explicit kill or server shutdown).` in your Proton log along with your video looks like the desktop environment or X is crashing, returning you to the login prompt. This might be a video driver issue. Are there any hints in `dmesg` or the system logs?

**Comment by SlugSleepy** [comment]
hi! what should i check exactly on system log ?
i did search on xorg log and syslog on today but nothing appear about steam, ghost of a tale or either x11.
[dmesg.log](https://github.com/ValveSoftware/Proton/files/9549806/dmesg.log)

i dont see any problem with dmesg too :cry: 

For the loging prompt, i did it myself, i disconnected my session.

**Comment by kisak-valve** [comment]
Kicking the user session out from underneath the game would explain that part of the log.

**Comment by SlugSleepy** [comment]
yeah i have no other choice, either i force logging out or i reboot my computer because the desktop become unusable.
the game window remains remain black in the foreground and i can't delock myself.

the game run well if i reinstall it, because the game run in windowed mode by default but as soon i select fullscreen, its the end.
I even thought it was the steam version that was the problem, so I tried to buy the game on a competitor merchant of steam with heroic but it makes the same problem
